WordPress/src/main/java/org/wordpress/android/ui/compose/components/IndeterminateCircularProgress.kt

```diff
@@ -0,0 +1,17 @@
+package org.wordpress.android.ui.compose.components
+
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+/**
+ * Indeterminate circular progress UI component. Used for operations that we don't how long are going to take.
+ */
+@Composable
+fun IndeterminateCircularProgress(
```

Comment:
I'm wondering how much value does a custom component like this add. Doesn't it make more sense to use the `CircularProgressIndicator` instead?

Reply:
Good point @planarvoid, I tend to agree with you, there's no real value in it since we can just use `CircularProgressIndicator` directly which makes the code cleaner imho.

I've applied this suggestion in [f30aa6f](https://github.com/wordpress-mobile/WordPress-Android/pull/16934/commits/f30aa6fdb28d45f86c8236018742ca22db77ff8b).